.tekton/osc-caa-pull-request.yaml

```
@@ -35,6 +35,10 @@ spec:
     value: cloud-api-adaptor/Dockerfile.openshift
   - name: path-context
     value: src
+  - name: prefetch-input
+    value: '[{"type": "rpm", "path": "./src/cloud-api-adaptor/"}, {"type": "gomod", "path": "./src/cloud-api-adaptor"}]'
```

Is there any reason this is not expanded as an array?

```suggestion
    value:
      - type: rpm
        path: ./src/cloud-api-adaptor/
      - type: gomod
        path: ./src/cloud-api-adaptor
```